playwright-e2e/playwright.config.ts

```diff
@@ -81,70 +91,29 @@ const testConfig: PlaywrightTestConfig = {
       fullPage: true
     },
     video: 'retain-on-failure', // Limit load on CI system because trace and video add load
-
-    userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36',
-    viewport: { width: 1280, height: 960 },
-    ignoreHTTPSErrors: true
-    // launchOptions: {
-    //   ignoreDefaultArgs: ['--hide-scrollbars'],
-    // },
   },
 
-  /* Configure projects for major browsers */
+  /* Configure projects for chromium browser */
   projects: [
     {
-      name: 'chromium',
-      // testMatch: '**/*.spec.ts',
-      grepInvert: /examples/,
-      use: {
-        browserName: 'chromium',
-        launchOptions: {
-          slowMo: 100,
-          // Account for minor difference in text rendering and resolution between headless and headed mode
-          ignoreDefaultArgs: ['--hide-scrollbars']
-        }
-      }
+      // Listing tests: npx playwright test --list --project="dsm"
+      // Running tests serially: npx playwright test --project="kit" --workers=1
+      name: 'dsm',
+      testDir: 'tests/dsm',
+      use: {}
+    },
+    {
+      name: 'dss',
+      testDir: 'tests',
+      testIgnore: ['dsm/**/*.spec.ts'],
+      use: {}
+    },
+    {
+      name: 'kit',
+      testDir: 'tests/dsm/kitUploadFlow',
+      fullyParallel: false,
+      use: {}
```

Comment:
Creating projects to run only DSM or DSS or Kit Upload tests. Example, `npx playwright test --project="dsm"`.
To run everything is unchanged, it is `npx playwright test`.